libafl/src/stages/mod.rs

```diff
@@ -71,13 +72,16 @@ pub mod generalization;
 pub mod generation;
 pub mod logics;
 pub mod power;
-pub mod stats;
 #[cfg(feature = "std")]
 pub mod sync;
 pub mod tracing;
 pub mod tuneable;
 #[cfg(feature = "unicode")]
 pub mod unicode;
+#[cfg(all(feature = "std", feature ="track_hit_feedbacks"))]
+pub use stats::{AflStatsStage, SyncTime, FuzzTime, CalibrationTime};
+#[cfg(all(feature = "std", feature ="track_hit_feedbacks"))]
```

Comment:
maybe not necessary to gatekeep everything behind `track_hit_feedbacks` here

Reply:
Yes it's not related to the feature at all(?)